### PR TITLE
FFM-8304: adds 797xx FM relnotes (STAGED)

### PR DESCRIPTION
**Pull request by StrawberryPad**

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Feature Flags release notes
 sidebar_label: Feature Flags
-date: 2023-06-14T10:00:25
+date: 2023-06-26T10:00:25
```
Comment (aimurphy) [inline release-notes/feature-flags.md:4]:
```suggestion
date: 2023-06-27T10:00:25
```

For now, deployment is on the 27th: https://harness.slack.com/archives/C012UCJARRN/p1687451440111309

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -17,14 +17,16 @@ Review the notes below for details about recent changes to Harness Feature Flags
 Harness deploys updates progressively to different Harness SaaS clusters. You can identify the cluster hosting your account in your Account Overview page. The features and fixes in the release notes may not be available in your cluster immediately.
 :::
 
-## Latest - June 22, 2023
+## Latest - June 26, 2023
```
Comment (aimurphy) [inline release-notes/feature-flags.md:20]:
```suggestion
## Latest - June 27, 2023
```

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -17,14 +17,16 @@ Review the notes below for details about recent changes to Harness Feature Flags
...
 
 ```mdx-code-block
 <Tabs>
   <TabItem value="What's new">
 ```
 
-This release does not include new features.  
+#### Feature Flags UI
+
+* On the **Feature Flags** page, when viewing the state of flags across all environments, the flag status column now scrolls all the rows as one. This allows for easier viewing of flag and environment states on one screen. (FFM-7643)
```
Comment (aimurphy) [inline release-notes/feature-flags.md:29]:
```suggestion
* On the **Feature Flags** page, when viewing the state of flags across all environments, the flag status column now scrolls all the rows as one. This makes it easier to view flag and environment states on one screen. (FFM-7643)
```

Optional suggestion.

**Review by aimurphy — DISMISSED** [review]
Optional wording suggestion and I think the date is now changed to the 27th (but of course that could change again).

**Comment by StrawberryPad** [comment]
Thanks @aimurphy

**Pull request merged**